docs/source/add_new_model.mdx

```diff
@@ -95,6 +95,24 @@ different formats - the model to a *pytorch_model.bin* file and the configuratio
 [`~PretrainedConfig.save_pretrained`], so that both model and configuration are saved.
 
 
+### Code style
+
+When coding your new model, keep in mind that Transformers is an opinionated library and we have a few quirks of our
+own regarding how code should be writte :-)
+
+1. The forward pass of your model should be fully written in the modeling file while being fully independent of other
+   models in the library. If you want to reuse a block from another model, copy the code and paste it with a
+   `# Copied from` comment on top (see [here](https://github.com/huggingface/transformers/blob/v4.17.0/src/transformers/models/roberta/modeling_roberta.py#L160)
+   for a good example).
+1. The code should be fully understandable, even by a non-native English speaker. This means you should pick
```

Comment:
```suggestion
2. The code should be fully understandable, even by a non-native English speaker. This means you should pick
```

Reply:
(looking at the lines below, I think this numbered list needs updated numbers 🙃  )